modules/azure-public-dns/dns-zones.tf

```diff
@@ -0,0 +1,12 @@
+resource "azurerm_resource_group" "resource_group" {
+  name = var.resource_group_name
+  location = "uksouth"
```

Comment:
probably best to use a variable for location. Since we will be going multi region hence the module should be able to cater to whatever location we pass to the module.